### PR TITLE
LPS-159837 | Add tests CanBeEnabled and WillNotDisplayWhenNotDefinedForSite

### PR DESCRIPTION
**Pull request by bronsonsoda**
**Test plan**
_WillNotDisplayWhenNotDefinedForSite_
Given there is no site-level JSON defined for current site
When instantiate the walkthrough
Then walkthrough is not instantiated on the site

_CanBeEnabled_
Given Walkthrough in System Settings
When walkthrough is enabled
Then success message appears

https://issues.liferay.com/browse/LPS-159837
https://issues.liferay.com/browse/LPS-159838

### CODE REVIEW

**Comment by bronsonsoda** [comment]
ci:test:frontend:nocompile

File: modules/apps/frontend-js/frontend-js-test/src/testFunctional/tests/WalkthroughSiteLevelConfiguration.testcase [hunk 1]
```diff
@@ -19,6 +19,37 @@ definition {
...
+
+			JSONLayout.addPublicLayout(
+				groupName = "Site Name",
+				layoutName = "Walkthrough Page 1");
+
+			JSONLayout.addWidgetToPublicLayout(
+				groupName = "Site Name",
+				layoutName = "Walkthrough Page 1",
+				widgetName = "JS Walkthrough Sample");
+		}
```
Comment (john-co) [inline modules/apps/frontend-js/frontend-js-test/src/testFunctional/tests/WalkthroughSiteLevelConfiguration.testcase:46]: Ah, I think what `walkthrough is enabled` means is just toggling the enable walkthrough on system settings and hitting submit button. And Then assert success message.

Comment (john-co) [reply modules/apps/frontend-js/frontend-js-test/src/testFunctional/tests/WalkthroughSiteLevelConfiguration.testcase:46]: Please update and force push.

**Comment by bronsonsoda** [comment]
ci:test:frontend:nocompile

**Comment by john-co** [comment]
Just started reviewing :)

**Comment by john-co** [comment]
ci:test:frontend:nocompile

**Comment by john-co** [comment]
ci:forward

**Pull request closed without merging**